src/test/java/com/epam/tc/hw3/pages/DifferentElementsPage.java

```diff
@@ -0,0 +1,58 @@
+package com.epam.tc.hw3.pages;
+
+import java.util.List;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class DifferentElementsPage extends BasePage {
+    @FindBy(xpath = "//li[@class='menu-title']")
```

Comment:
Let's rewrite this in css form
```suggestion
    @FindBy(css = "li.menu-title")
```

\+ `logs` in the same way

Reply:
it is ready